app/data/models/Deck.py

```diff
@@ -0,0 +1,29 @@
+from app.data.repositories.UsersRepository import user
+from app.data.repositories import CardsRepository
+
+class Deck:
+
+
+    def __init__(self, deck_id, user_id, name):
+        
+        self.deck_id = deck_id
+        self.user_id = user_id
+        self.name = name
+
+
+    @classmethod
+    def new_deck(cls, user_id, name):
+        return cls(None, user_id, name)
+
+
+    @classmethod
+    def deck(cls, args):
+        return cls(*args)
+
+
+    def getDecksCards(self):
+
+        cards = CardsRepository.getCardsByDeckId(self.deck_id)
```

Comment:
return CardsRepository.getCardsByDeckId(self.deck_id) ?

Reply:
yes :D is it wrong to use methods from a different repository?

Reply:
i dont think so, just you can do it in one line